frontend/src/component/integrations/JiraIntegration/JiraIntegration.tsx

```diff
@@ -0,0 +1,114 @@
+    backgroundColor: theme.palette.grey[100],
+    borderRadius: theme.shape.borderRadiusLarge,
+    padding: theme.spacing(3),
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(1),
+}));
+
+export const StyledIconLine = styled('p')(({ theme }) => ({
+    display: 'flex',
+    marginLeft: theme.spacing(1),
+    alignItems: 'center',
+}));
+
+export const StyledLink = styled('a')({
+    textDecoration: 'none',
+});
+
+export const JIRA_INFO = {
+    name: 'jira',
+    displayName: 'Jira',
+    description:
+        'Create, connect, manage, and approve Unleash feature flags directly from Jira',
+    documentationUrl:
+        'https://docs.getunleash.io/reference/integrations/jira-cloud-plugin-installation',
+};
+
+export const JiraIntegration = () => {
+    const { name, displayName, description, documentationUrl } = JIRA_INFO;
+
+    return (
+        <FormTemplate
+            title={`${displayName}`}
+            description={description || ''}
+            documentationLink={documentationUrl}
+            documentationLinkLabel="Jira documentation"
+        >
+            <StyledContainer>
+                <StyledGrayContainer>
+                    <StyledIconLine>
+                        <IntegrationIcon name={name} /> How does it work?
+                    </StyledIconLine>
+                    <ul>
+                        <li>
+                            Create a new feature flag directly within Jira, or
+                            connect existing flags to any Jira issue.
+                        </li>
+                        <li>
+                            Keep track of your flag status for each environment.
+                        </li>
+                        <li>
+                            Activate/deactivate feature flags directly within
+                            Jira.
+                        </li>
+                        <li>
+                            Initiate change requests when guarded flags are
+                            activated/deactivated within Jira.
+                        </li>
+                    </ul>
+                </StyledGrayContainer>
+                <StyledGrayContainer>
+                    <StyledLink
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        href="https://marketplace.atlassian.com/apps/1231447/unleash-enterprise-for-jira"
+                    >
+                        View plugin on Atlassian marketplace
+                    </StyledLink>
+                </StyledGrayContainer>
+                <Divider />
+                <JiraImageContainer
+                    title={'Manage your feature flags for each environment'}
+                    description={
+                        'View your feature flag status for each of your environments. Quickly turn features on and off directly within Jira.'
+                    }
+                    src={manage}
```

Comment:
Yeah, based on my understanding of the image container now, I think this would be better served as a [figure with a figcaption element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure). That fig caption takes the place of the alt text in this case too, so you don't need to think about that.

Reply:
@thomasheartman changed to figure. I am not sure if title and description both should belong to figcaption, but based on googleing I think yes.

Reply:
Hmm, title in the figcaption? Yeah, it's permitted content, so I think that makes sense 😄